### PR TITLE
HAFI

### PR DESCRIPTION
**Pull request by Harkerfield**
PR request for reserve. not finished...

### CODE REVIEW

File: client/src/Reserve/UserDropDown.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                            </Listbox.Option>
+                        ))}
+                    </Listbox.Options>
+                </Transition>
+            </div>
+        </Listbox>
+    </>)
+}
+
+export default UserDropDown;
```
Comment (JacobTheEldest) [inline client/src/Reserve/UserDropDown.js:64]: missing newline

**Review by hackek — APPROVED** [review]
Self-contained changes and the seed change makes sense. Approved

File: package-lock.json
Comment (JacobTheEldest) [inline package-lock.json:1]: This file shouldn't exist. npm install in the wrong directory?

File: package.json
Comment (JacobTheEldest) [inline package.json:1]: This file shouldn't exist. npm install in the wrong directory?

**Review by JacobTheEldest — CHANGES_REQUESTED** [review]
Please address comments.

**Review by JasonMartin85 — APPROVED** [review]
Looks good to me, mostly new stuff and formatting things better

**Review by JacobTheEldest — APPROVED** [review]

**Pull request merged**